app/forms/webauthn_setup_form.rb

```diff
@@ -172,6 +174,7 @@ def extra_analytics_attributes
       pii_like_keypaths: [[:mfa_method_counts, :phone]],
       authenticator_data_flags: authenticator_data_flags,
       unknown_transports: invalid_transports.presence,
+      aaguid: aaguid,
```

Comment:
Can we add corresponding YARDoc updates for method documentation in `analytics_events.rb`?

I would have also expected this to be enforced by [`UndocumentedParamsChecker`](https://github.com/18F/identity-idp/blob/9d75dd33c8e9a05960605c6e9d3d525c97048830/spec/support/fake_analytics.rb#L73) and fail the build, which makes me worried we might not have full coverage for the code which would log this event.